Documentation/ceph-teardown.md

```diff
@@ -59,6 +59,21 @@ In the future this step will not be necessary when we build on the K8s local sto
 
 If you modified the demo settings, additional cleanup is up to you for devices, host paths, etc.
 
+Disks on nodes used by Rook for osds can be reset to a usable state with the following methods:
+```sh
+#!/usr/bin/env bash
+DISK="/dev/sdb"
+# Zap the disk to a fresh, usable state (zap-all is important, b/c MBR has to be clean)
+# You will have to run this step for all disks.
+sgdisk --zap-all $DISK
+
+# These steps only have to be run once on each node
+# If rook sets up osds using ceph-volume, teardown leaves some devices mapped that lock the disks.
```

Comment:
it seems a bit easier to vgremove them before sgdisk zap

Reply:
`vgdisplay` reports no volume groups existing, so I don't believe that is an option.

Reply:
right, because lvm was created by the prepare job which has no contact with lvmetad. we need to run `vgscan` first.

Reply:
Just confirmed that `vgscan` doesn't result in anything in the `vgdisplay` output.